### PR TITLE
Added additional technical edits for Partner Manager 2.9

### PR DESCRIPTION
**Pull request by dhirsch1**

### CODE REVIEW

File: modules/ROOT/pages/file-name-pattern.adoc [hunk 1]
```diff
@@ -23,26 +23,29 @@ The following are the supported property placeholders and the dynamic value the
 |${sss} |Second: for example, `46`
 |${timeInMillis} |Milliseconds: for example, `345`
 |${ICN} |Interchange Control Number
+|${ICN} |Group Control Number
```
Comment (vijayang) [inline modules/ROOT/pages/file-name-pattern.adoc]: Please change ${ICN} to ${GCN}

File: modules/ROOT/pages/file-name-pattern.adoc [hunk 2]
```diff
@@ -23,26 +23,29 @@ The following are the supported property placeholders and the dynamic value the
...
 
 == Examples
 
+[%header%autowidth.spread]
 |===
 |File Name Pattern |Generated File Name
 |${DocType}_${ISASender}_${YYYYMMDD}_${HHmmSSsss}_${ICN}.edi |`850_NTORETAIL_20201102_192655521_300001.edi`
-|INVOICE_${SenderGSID}_${ICN}.${formatTypeExt}
+|INVOICE_${SenderGSID}_${ICN}.${FormatTypeExt}
 |`INVOICE_MYTHICALSUPPLIERS_12355343.XML`
```
Comment (vijayang) [inline modules/ROOT/pages/file-name-pattern.adoc]: Please change "XML" to "xml"

File: modules/ROOT/pages/file-name-pattern.adoc [hunk 3]
```diff
@@ -23,26 +23,29 @@ The following are the supported property placeholders and the dynamic value the
...
 |${uuid}.edi
 |`93de0793-63e1-5a06-aedf-fcd052ed2155.edi`
 |${DocType}-${YYYY}-${MM}-${DD}-${HHmmSSsss}.dat
 |`204-2020-06-24-192655521.dat`
-|${DocType}_${ISASender}_${YYYYMMDD}_${HHmmSSsss}_${ICN}.${formatTypeExt}
+|${DocType}_${ISASender}_${YYYYMMDD}_${HHmmSSsss}_${ICN}.${FormatTypeExt}
 |`850_NTORETAIL_20201102_192655521_300001.JSON`
```
Comment (vijayang) [inline modules/ROOT/pages/file-name-pattern.adoc]: Please change "JSON" to "json"

**Review by vijayang — CHANGES_REQUESTED** [review]

**Review by vijayang — DISMISSED** [review]

File: modules/ROOT/pages/setup-payload-storage-API.adoc [hunk 4]
```diff
@@ -43,6 +43,14 @@ Your payload storage API must be accessible over the internet via HTTPS. Ensure
 * *Endpoint name*: The name of the payload storage API
 * *Description*: The optional description for your payload storage
 * *URL*: The path to your payload storage
++
+This URL does the following:
++
+** Stores payloads when transactions process in the runtime
+** Retrieves the payload when you click *View Payload* on the *Transaction Activity* page, unless a payload view URL is configured
++
+* *Payload view URL*: (Optional) A secondary payload storage API URL that you can use to view the payload on the Activity page if the payload storage API is restricted to your internal network.
```
Comment (sathya0) [inline modules/ROOT/pages/setup-payload-storage-API.adoc]:
```suggestion
* *Payload view URL*: (Optional) A secondary payload storage API URL that you can use to view the payload on the *Activity* page if the payload storage API is restricted to your internal network.
```

Comment (sathya0) [inline modules/ROOT/pages/setup-payload-storage-API.adoc]:
```suggestion
* *Payload view URL*: (Optional) A secondary payload storage API URL that you can use to view the payload on the Activity page if the payload storage API is restricted to your internal network
```

**Review by sathya0 — DISMISSED** [review]

**Review by sathya0 — APPROVED** [review]

**Pull request merged**